src/utils.py

```diff
@@ -0,0 +1,96 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.exc import SQLAlchemyError
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+import requests
+import os
+import logging
+import json
+import datetime
+import semantic_version as sv
+
+from f8a_worker.models import OSIORegisteredRepos
+
+logger = logging.getLogger(__name__)
+
+GREMLIN_SERVER_URL_REST = "http://{host}:{port}".format(
+    host=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_HOST", "localhost"),
+    port=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_PORT", "8182"))
+
+LICENSE_SCORING_URL_REST = "http://{host}:{port}".format(
+    host=os.environ.get("LICENSE_SERVICE_HOST"),
+    port=os.environ.get("LICENSE_SERVICE_PORT"))
+
+
+# Create Postgres Connection Session
+class Postgres:
+    def __init__(self):
+        self.connection = 'postgresql://{user}:{password}@{pgbouncer_host}:{pgbouncer_port}' \
+                          '/{database}?sslmode=disable'. \
+            format(user=os.getenv('POSTGRESQL_USER'),
+                   password=os.getenv('POSTGRESQL_PASSWORD'),
+                   pgbouncer_host=os.getenv('PGBOUNCER_SERVICE_HOST', 'bayesian-pgbouncer'),
+                   pgbouncer_port=os.getenv('PGBOUNCER_SERVICE_PORT', '5432'),
+                   database=os.getenv('POSTGRESQL_DATABASE'))
+        engine = create_engine(self.connection)
+
+        self.Session = sessionmaker(bind=engine)
+        self.session = self.Session()
+
+    def session(self):
+        return self.session
+
+
+_rdb = Postgres()
+_session = _rdb.session
+
+
+def get_osio_user_count(ecosystem, name, version):
```

Comment:
@GeetikaBatra please remove this function

Reply:
sure, will remove it.